content/java/concepts/strings/terms/lastIndexOf/lastIndexOf.md

```diff
@@ -0,0 +1,38 @@
+---
+Title: '.lastIndexOf()' 
+Description: 'Searches a string for a specified value, and returns the position of the match.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'String'
+  - 'Method'
+  - 'Characters'
+CatalogContent:
+  - 'learn-Java'
+  - 'paths/computer-science'
+---
+
+The **`.lastIndexOf()`** method is a method that searches a string for a specified value, and returns the position of the match. It searches the string from the end to the beginning, and returns the index within the string of the last occurrence of the specified value, or -1 if the value is not found.
+
+## Syntax
+
+```pseudo
+lastIndexOf(String searchValue, int fromIndex)

```

Comment:
Let's include the string that the method is called on here. See this example: https://www.codecademy.com/resources/docs/java/strings/codePointBefore